src/main/fc/config.c

```diff
@@ -548,7 +548,8 @@ void validateAndFixConfig(void)
 {
 #if !defined(USE_UNCOMMON_MIXERS) && !defined(USE_QUAD_MIXER_ONLY) && !defined(USE_OSD_SLAVE)
     mixerMode_e mixerMode = mixerConfigMutable()->mixerMode;
-    if (mixerMode != MIXER_CUSTOM && mixerMode != MIXER_CUSTOM_AIRPLANE && mixerMode != MIXER_CUSTOM_TRI && mixers[mixerMode].motor == NULL) {
+
+    if (mixerMode != MIXER_CUSTOM && mixerMode != MIXER_CUSTOM_AIRPLANE && mixerMode != MIXER_CUSTOM_TRI && mixerMode != MIXER_GIMBAL && mixerMode != MIXER_PPM_TO_SERVO && mixers[mixerMode].motor == NULL) {
```

Comment:
> This is getting out of hand ... maybe customness of mixer should be defined somewhere

I think this part of the code would become legacy soon?

> BTW: what about MIXER_HELI_120_CCPM?

I couldn't find any code that supports `MIXER_HELI_*`. Is there any?

Oh yes, and `MIXER_PPM_TO_SERVO` also.